### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by kimitaWanjohi**
**Made the page responsive on a desktop screen by:**

- Adding media queries for 768px and 1024px.
- Styled section to fit the desktop screen.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,57 +10,77 @@
...
-        <section class="headline section">
-            <div class="p-2 container">
-                <h1 class="main-title">Hi There. I'm Kimita</h1>
-                <h2 class="sub-title">I'm a software developer</h2>
-                <p class="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                    don’t hestiate to contact me.
-                </p>
-                <div class="socials">
+        <section class="headline">
+            <!-- mobile nav -->
```
Comment (tufoinnkuo10) [inline index.html:14]: - [ ] kindly add some space between your headline  section and the navigation bar so it follows the original design
 yours
Screenshot (830)_LI
figma
Screenshot (831)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @kimitaWanjohi 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ great design

Required Changes ♻️
Check the comments under the review
- [ ] please remove the live demo section in your readme file since you don't yet have a deployed version of your application. this will make your readme more professional

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 2]
```diff
@@ -277,24 +281,29 @@ <h1 class="about-title">
...
-                    <div class="mb-2">
-                        <textarea rows="4" cols="auto" maxlength="500" id="message" name="message" required class="form-control w-full" placeholder="Enter text here"></textarea>
-                    </div>
-                    <div class="mb2">
-                        <button class="form-btn">Get in touch</button>
-                    </div>
-                </form>
+                <div class="lg-grid">
+                    <h1 class="main-title">
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
```
Comment (osamaashraf6) [inline index.html:286]: - [ ] Kindly 😃 as stated by the previous reviewer please try as much as possible to follow the Figma design. kindly align the text and form of the contact section with the cards above it so it follows the Figma design
- [ ]  please kindly add the email field like in the original design. you can use one input field for the name if you wish( you can opmit the last name and use one input field for the name) but the email field should appear like in the original design.💯
- [ ] please 😊 the background image should be visible around the contact section like in the original design. remember you are practicing how to deliver what your customer will ask for in the future so the little details will matter. nice work.


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| image   |    image   |

File: index.html [hunk 3]
```diff
@@ -277,24 +281,29 @@ <h1 class="about-title">
...
+                    
+                    <form class="w-full flex justify-start flex-col p-2 mt-2" action="https://formspree.io/f/xlekblnd" method="POST">
+                        <div class="flex gap-2">
+
+                            <div class="mb-2 w-full">
+                                <input type="text" name="name" maxlength="30" required id="name" class="form-control w-full" placeholder="Full name">
+                            </div>
+                            <div class="mb-2 w-full">
+                                <input type="email" name="email" required id="email" class="form-control w-full" placeholder="Email Address">
+                            </div>
```
Comment (osamaashraf6) [inline index.html:297]: - [ ] [optional] please 😊 remember, you are supposed to stick to the design as much as possible. your mobile version contact form is different from the Figma version. the email under the social links is missing as well. kindly fix that.
yours💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  image   |    image |

File: index.html (hunk 1 above)
Comment (osamaashraf6) [inline index.html:14]: - [ ] Kindly 😃 as stated by the previous reviewer kindly add some space between your headline section and the navigation bar so it follows the original design yours

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  image  |  image  |

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi  @kimitaWanjohi ☺️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

you-are-near

## Highlights ✔️

- [x] Good commit messages.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow

 ## Required Changes ♻️

- kindly 😊 as stated by the previous reviewer, please remove the live demo section in your readme file since you don't yet have a deployed version of your application. this will make your readme more professional
- Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
Hi @kimitaWanjohi  , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**